modules/upgrade-upgrading.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-aws.adoc
+
+[id='upgrade-upgrading-{context}']
+= Updating a cluster
+
+If updates are available, you can update your cluster from the web console.
+
+.Prerequisites
+
+* Access to the web console as a user with `admin` privileges.
```

Comment:
Should we link to the section that defines how to give a user admin privileges?

Reply:
I'm adding a link to the assembly - we currently cannot add internal links to individual modules due to tooling constraints.